scripts/prettify

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+stack exec -- brittany --indent=2 --write-mode=inplace src/*.hs;
+stack exec -- brittany --indent=2 --write-mode=inplace src/*/*.hs;
+stack exec -- brittany --indent=2 --write-mode=inplace test/*.hs;
+stack exec -- brittany --indent=2 --write-mode=inplace test/*/*.hs;
```

Comment:
What's the difference between `lint` / `prettify` here?

Reply:
Pretty much the same as between prettier and eslint.